cpp/models/abm/person.h

```diff
@@ -479,15 +470,14 @@ class Person
     Mask m_mask; ///< The Mask of the Person.
     std::vector<ScalarType>
         m_compliance; ///< Vector of compliance values for all #InterventionType%s. Values from 0 to 1.
-    LocalIndex m_person_index; ///< LocalIndex of the Person. Corresponds to the index in m_persons in Model.
     std::vector<uint32_t> m_cells; ///< Vector with all Cell%s the Person visits at its current Location.
     mio::abm::TransportMode m_last_transport_mode; ///< TransportMode the Person used to get to its current Location.
     Counter<uint32_t> m_rng_counter{0}; ///< counter for RandomNumberGenerator.
     CustomIndexArray<TestResult, TestType> m_test_results; ///< CustomIndexArray for TestResults.
     std::vector<int>
         m_assigned_location_model_ids; ///< Vector with model ids of the assigned locations. Only used in graph abm.
-    GlobalID
-        m_global_id; ///< Unique identifier of a person. Is only relevant in graph abm, otherwise is equal to m_person_index.
+    PersonId m_person_id; ///< Unique identifier of a person.
+    uint32_t m_rng_index;
```

Comment:
Ah, I see. Well now instead of the person index we have the rng index.

In all current use cases, we could use the other constructor of PersonalRNG that uses the index directly from its index in the world. However, when we want to construct the PersonalRNG inside the Person or anywhere where the world is not known, this is a problem.

Further, this rng_index needs to be updated each time a person is removed and added to a world, such that it uses the correct world index and doesn't clash with the same rng_index of another person.

Let's discuss in the next meeting.

Reply:
We decided to save all information in the person to keep access to the same RNG, even after the person moved between models.